MarsRover.Tests/FileInputShould.cs

```diff
@@ -36,11 +38,9 @@ public void ReturnFilePath_GivenValidFilePath(string path)
         [InlineData("TestFile/testFile2.rtf")]

```

Comment:
These could be more "obviously invalid" e.g. `bad-extension.rtf`, and the valid cases could be more "obviously valid" e.g. `valid-file-1.txt`